dask/dataframe/core.py

```diff
@@ -450,6 +451,122 @@ def _get_binary_operator(cls, op, inv=False):
                                                 columns=self._elemwise_cols)
 
 
+    def _aca_agg(self, token, func, aggfunc=None):
+        """ Wrapper for aggregations """
+        raise NotImplementedError
+
+    @wraps(pd.DataFrame.sum)
+    def sum(self, axis=None):
+        axis = self._validate_axis(axis)
+        if axis == 1:
+            f = lambda x: x.sum(axis=1)
+            return self.map_partitions(f, None)
+        else:
+            return self._aca_agg(token='sum', func=lambda x: x.sum())
+
+    @wraps(pd.DataFrame.max)
+    def max(self, axis=None):
+        axis = self._validate_axis(axis)
+        if axis == 1:
+            f = lambda x: x.max(axis=1)
+            return self.map_partitions(f, None)
+        else:
+            return self._aca_agg(token='max', func=lambda x: x.max())
+
+    @wraps(pd.DataFrame.min)
+    def min(self, axis=None):
+        axis = self._validate_axis(axis)
+        if axis == 1:
+            f = lambda x: x.min(axis=1)
+            return self.map_partitions(f, None)
```

Comment:
Perhaps `map_partitions` should accept and pass through keyword arguments.  E.g.

    return `self.map_partitions(pd.DataFrame.min, axis=1)` 

Lambdas may cause troubles with `tokenize`.